UIKitDZMain/ViewController.swift

```diff
@@ -7,10 +7,11 @@
 
 import UIKit
 
+///  Светофор сториборд
 final class ViewController: UIViewController {
```

Comment:
```suggestion
final class LighterViewController: UIViewController {
```
Переименуй контроллер или можно удалить его, если не используешь